cogs/logcog.py

```diff
@@ -0,0 +1,40 @@
+from discord.ext import commands
+import discord
+import json
+
+class LogCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    @commands.has_role("Bot Commander")
+    async def setlogchannel(self, ctx, channel):
+        channel_obj = discord.utils.get(ctx.guild.text_channels, name=channel)
```

Comment:
We use so many emojis in our channel names. Using IDs instead of names in admin-only commands is probably preferable.